README.md

```diff
@@ -52,6 +52,10 @@ with line-by-line highlighting:
 pytest --cov-report html:htmlcov
 ```
 
+## Dependencies
+
+External dependencies are listed in `requirements.txt` and can be installed with `pip` as outlined in the [Installation instructions](#installation-instructions) or manually. The dependencies on `beautifulsoup4`, `lxml` and `html5lib` are necessary for [`pandas`](https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies). Developers additionally need [`pytest`](https://docs.pytest.org) and are encouraged to use tools such as [`pylint`](https://pylint.readthedocs.io), [`pycodestyle`](http://pycodestyle.pycqa.org) [`pydocstyle`](http://www.pydocstyle.org) and [`yapf`](https://github.com/google/yapf) for proper code formatting.
```

Comment:
Does anyone have any opinions on where we break lines in the README and other text files? At 80 characters, every sentence, every paragraph?

Reply:
Good point. I would say 80 characters it is as for code, just to be consistent. Let me fix that, also the Copyright notice misses these newlines.

Reply:
Some of the links are more than 80 symbols, so I'm actually not sure how to do that. Any thoughts? Just let `[](http://)` excced the line?

Reply:
Actually it seems possible to break urls: https://stackoverflow.com/questions/19985235/break-long-lines-in-markdown-code. Let me just break them or is that bad practice when someone wants to actually copy the url from the source code?

Reply:
It looks weird. 
```
## Dependencies

External dependencies are listed in `requirements.txt` and can be installed 
with `pip` as outlined in the [Installation instructions
](#installation-instructions) or manually. The dependencies on 
`beautifulsoup4`, `lxml` and `html5lib` are necessary for [`pandas`
](https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies).
Developers additionally need [`pytest`](https://docs.pytest.org) and are 
encouraged to use tools such as [`pylint`](https://pylint.readthedocs.io),
[`pycodestyle`](http://pycodestyle.pycqa.org), [`pydocstyle`](
http://www.pydocstyle.org) and [`yapf`](https://github.com/google/yapf)
for proper code formatting.
```

Maybe just add line breaks every sentence or paragraph would make more sense. Any thoughts?

Reply:
We could use markdown [references](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links):

```markdown
## Dependencies

External dependencies are listed in `requirements.txt` and can be installed
with `pip` as outlined in the  [Installation instructions][0] or manually.
The dependencies on `beautifulsoup4`, `lxml` and `html5lib` are necessary for
[pandas][1]. Developers additionally need [`pytest`][2] and are encouraged to
use tools such as [`pylint`][3], [`pycodestyle`][4], [`pydocstyle`][5] and
[`yapf`][6] for proper code formatting.

[0]: #installation-instructions
[1]: https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies
[2]: https://docs.pytest.org
[3]: https://pylint.readthedocs.io
[4]: http://pycodestyle.pycqa.org
[5]: http://www.pydocstyle.org
[6]: https://github.com/google/yapf
```

Which renders on github with the soft breaks being registered as line breaks [which shouldn't be possible without extra spaces at the end of the line...](https://gist.github.com/shaunlebron/746476e6e7a4d698b373) :shipit:

**Maybe this doesn't happen in the markdown rendering for `md` files**

## Dependencies

External dependencies are listed in `requirements.txt` and can be installed 
with `pip` as outlined in the  [Installation instructions][0] or manually.
The dependencies on `beautifulsoup4`, `lxml` and `html5lib` are necessary for
[pandas][1]. Developers additionally need [`pytest`][2] and are encouraged to
use tools such as [`pylint`][3], [`pycodestyle`][4], [`pydocstyle`][5] and
[`yapf`][6] for proper code formatting.

[0]: #installation-instructions
[1]: https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies
[2]: https://docs.pytest.org
[3]: https://pylint.readthedocs.io
[4]: http://pycodestyle.pycqa.org
[5]: http://www.pydocstyle.org
[6]: https://github.com/google/yapf

Reply:
How the hell did you manage to render that? Looks good.